### ISSUE TITLE
[Bug]: Doesn't do anything

### ISSUE THREAD

**Issue by jnpatrick99** [continue-revolution/sd-webui-animatediff#9]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### Have you read FAQ on README?

- [X] I have updated WebUI and this extension to the latest version

### What happened?

It doesn't do anything, I mean it outputs GIF but it's only 1 frame. 
Seems no errors are displayed. All other plugins work correctly, I set Video frame number to 16, Frames per second 8

### Steps to reproduce the problem

1. Go to UI
2. Enable AnimateDiff
3. Press Generate

### What should have happened?

Animated GIF

### Commit where the problem happens

version: [v1.4.0] •  python: 3.10.7  •  torch: 2.0.0+cu118  •  xformers: 0.0.20  •  gradio: 3.32.0

### What browsers do you use to access the UI ?

Google Chrome

### Command Line Arguments

```Shell
--listen --port 7861  --skip-version-check --skip-torch-cuda-test --skip-python-version-check --opt-split-attention-v1 --xformers --no-half-vae --api --vae-path .\\payload\\vae-ft-mse-840000-ema-pruned.ckpt --enable-insecure-extension-access
```

*I tried removing these args it didn't help.

### Console logs

```Shell
2023-07-18 17:25:06,318 - AnimateDiff - INFO - Removing motion module from SD1.5 UNet input blocks.01<00:00, 17.91it/s]
2023-07-18 17:25:06,318 - AnimateDiff - INFO - Removing motion module from SD1.5 UNet output blocks.
2023-07-18 17:25:06,318 - AnimateDiff - INFO - Removal finished.
2023-07-18 17:25:06,319 - AnimateDiff - INFO - Merging images into GIF.
2023-07-18 17:25:06,380 - AnimateDiff - INFO - AnimateDiff process end.
```

### Additional information

Windows 10, RTX 3090

**Comment by JenXIII** [comment]
Open it in an image editor that supports .gif, you'll probably find the frame times are on the order of 100s.  This can be solved by editing animatediff.py line 153 so it reads duration=(1/fps) instead of duration=(1000/fps).  I think maybe it expects milis in the dev's environment but in ours it expects seconds in this function.  Just a guess though.

**Comment by JPW0080** [comment]
remove the --xformers flag.

**Comment by IntellectzProductions** [comment]
I keep getting this error. It stops at the grid then:
    Traceback (most recent call last):
      File "E:\stable-diffusion-webui\modules\call_queue.py", line 55, in f
        res = list(func(*args, **kwargs))
      File "E:\stable-diffusion-webui\modules\call_queue.py", line 35, in f
        res = func(*args, **kwargs)
      File "E:\stable-diffusion-webui\modules\txt2img.py", line 57, in txt2img
        processed = processing.process_images(p)
      File "E:\stable-diffusion-webui\extensions\sd-webui-prompt-history\lib_history\image_process_hijacker.py", line 25, in process_images
        global_state.add_config( uuid.uuid4().hex, res.prompt[:64], shared.opts.sd_model_checkpoint, res.infotexts[0], res.images[0])
      File "E:\stable-diffusion-webui\extensions\sd-webui-prompt-history\scripts\prompt_history_script.py", line 76, in add_config
        new_height = int(new_width * img.height / img.width)
    AttributeError: 'str' object has no attribute 'height'

**Comment by continue-revolution** [comment]
@JenXIII Yes, this problem exists in the previous version, but is fixed in [v1.1.0](https://github.com/continue-revolution/sd-webui-segment-anything/releases/tag/v1.1.0)

@JPW0080 You can add xformers in [v1.1.0](https://github.com/continue-revolution/sd-webui-segment-anything/releases/tag/v1.1.0), but xformers will not be applied to motion modules

**Comment by continue-revolution** [comment]
Again, please update your extension and re-try. If the problem still exists then let me know.

**Comment by jnpatrick99** [comment]
Thank you, I was able finally to run it by installing webui from scratch in a different folder.
 
Also I noticed this extension doesn't work on any other older webui except the most recent one from GitHub. Is it intentional? I prefer using older versions sometimes as webui tend to break things from time to time. Thank you.

**Comment by 0m3ga4** [comment]
> Again, please update your extension and re-try. If the problem still exists then let me know.

Not OP, but having this exact issue as of right now on fresh install.

**Comment by jstoeffler** [comment]
I have the same problem I think. When I click "Generate" Automatic1111 just runs the normal image generation, doesn't run the script, no AnimateDiff logs.

It seems that in my case the variable `enable_animatediff` is False, even though I checked the checkbox. I came to this conclusion because when I comment [this line](https://github.com/continue-revolution/sd-webui-animatediff/blob/bcca007a1ba700f1cd231698c00e460a63581171/scripts/animatediff.py#L134) out, then something happens (it starts logging, and renders many images).

Not sure where the problem comes from, and how this variable is passed to the plugins function.
